docs/docs/recipes/integrate-logto/README.md

```diff
@@ -4,28 +4,38 @@ sidebar_position: 1
 
 # 🔗 Integrate Logto in your application
 
-We provide a bunch of SDKs to let you integrate Logto with front-end clients with ease. If you are using traditional web technology (i.e., server directly renders frontend), please check out our traditional web guide.
+We provide a bunch of SDKs to let you integrate Logto with your clients with ease. If the list doesn't cover your desired platform / framework, please file a feature request or contribute a new SDK.
 
-If the list doesn't cover your desired platform/framework, please file a feature request or contribute a new SDK.
+## Native App
 
-## SPA
+An app that runs in a native environment.
+
+- [iOS](./ios.mdx)
+- [Android](./android.mdx)
+
+Usually, Native apps need to talk to an API (resource). Please refer to [Protect your API](../protect-your-api/README.mdx) for authenticating in your API.
+
+## Single Page App
+
+An app that runs in a web browser and dynamically updates data in place.
 
 - [Vanilla JS](./vanilla-js.mdx)
 - [React](./react.mdx)
 - [Vue](./vue.mdx)
 
+Usually, SPAs need to talk to an API (resource). Please refer to [Protect your API](../protect-your-api/README.mdx) for authenticating in your API.
+
 ## Traditional Web
 
-- [Traditional web](./traditional.mdx)
+An app that renders and updates pages by the web server alone. E.g., server-side rending.
+
+- [Traditional Web general guide](./traditional.mdx)
 - [Next.js](./next-js.mdx)
 - [Express](./express.mdx)
 - [Go](./go-web.mdx)
 
-## Native
-
-- [iOS](./ios.mdx)
-- [Android](./android.mdx)
+## Machine to Machine
 
-## Backend Application
+An app (usually without UI) that directly talks to resources.
```

Comment:
nit: do we need to add the example of Logto's management API, this is the most common case.

Reply:
good call, i'll update later

Reply:
updated, added some specific content about accessing Mgmt API, should be good now.